tool/tctl/common/helpers_test.go

```diff
@@ -201,9 +201,9 @@ func withFileDescriptors(fds []service.FileDescriptor) testServerOptionFunc {
 	}
 }
 
-func withFakeClock() testServerOptionFunc {
+func withFakeClock(clock time.Time) testServerOptionFunc {
```

Comment:
You should pass in the fake clock, not a time. It's called with fake _clock_, after all 😉

Reply:
Hah good call! Done.